### PR TITLE
Small readme and defaults fix

### PR DESCRIPTION
**Pull request by DACUS1995**
Two small fixes:
- A variable renamed in readme output -> response
- The default parameter for a list.

### CODE REVIEW

File: codeinterpreterapi/session.py [hunk 1]
```diff
@@ -167,10 +167,13 @@ async def output_handler(self, final_response: str) -> CodeInterpreterResponse:
     async def generate_response(
         self,
         user_msg: str,
-        files: list[File] = [],
-        detailed_error: bool = False,
+        files: Optional[list[File]] = None,
+        detailed_error: Optional[bool] = False,
     ) -> CodeInterpreterResponse:
         """Generate a Code Interpreter response based on the user's input."""
+        if files is None:
```
Comment (shroominic) [inline codeinterpreterapi/session.py:174]: this does the same by adding more lines

File: README.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ async def main():
     await session.astart()
 
     # generate a response based on user input
-    output = await session.generate_response(
+    response = await session.generate_response(
```
Comment (shroominic) [inline README.md:39]: lgtm thanks

**Review by shroominic — CHANGES_REQUESTED** [review]
README fixes are good but session fixes not necessary

File: codeinterpreterapi/session.py (hunk 1 above)
Comment (DACUS1995) [reply codeinterpreterapi/session.py]: Ok, reverted the change. Indeed in this case the list doesn't seem to change after the call, but from my experience, you can never be too safe with mutable variables :).

File: codeinterpreterapi/session.py [hunk 3]
```diff
@@ -167,8 +167,8 @@ async def output_handler(self, final_response: str) -> CodeInterpreterResponse:
     async def generate_response(
         self,
         user_msg: str,
-        files: list[File] = [],
-        detailed_error: bool = False,
+        files: Optional[list[File]] = [],
+        detailed_error: Optional[bool] = False,
```
Comment (shroominic) [inline codeinterpreterapi/session.py:171]: I don't think we need to accept None values here

**Review by shroominic — COMMENTED** [review]
This already got resolved through another PR but still thanks for pointing this out!

**Pull request closed without merging**